src/main/java/TextParser.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Represent a class to handle the parsing of the text for processing of commands.
+ */
+public class TextParser {
+
+    /**
+     * Parse a string into array using String.split(" ", 2).
+     * @param line to be parsed
+     * @return String[]
+     */
+    public static String[] myFirstParser(String line) {
```

Comment:
Could use a more meaningful names or even merge with your other parsers and add a new parameter to indicate the regular expression.